packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -25,10 +30,40 @@ const component = (): ViewTemplate => html`
     <${richTextEditorTag}></${richTextEditorTag}>
 `;
 
+const editorSizingTestCase = (
+    [widthLabel, widthStyle]: [string, string],
+    [heightLabel, heightStyle]: [string, string]
+): ViewTemplate => html`
+    <p style="font: var(${cssPropertyFromTokenName(
+        tokenNames.bodyFont
+    )}); margin-bottom: 0px;">${widthLabel}; ${heightLabel}</p>
+    <div style="width: 500px; height: 150px; outline: 1px dotted black;">
+        <${richTextEditorTag} style="${widthStyle}; ${heightStyle};">
+            <${buttonTag} slot="footer-actions" appearance="ghost">Cancel</${buttonTag}>
+            <${buttonTag} slot="footer-actions" appearance="outline">Ok</${buttonTag}>
+        </${richTextEditorTag}>
+    </div>
+`;
+
 export const richTextEditorThemeMatrix: StoryFn = createMatrixThemeStory(
     createMatrix(component)
 );
 
+export const richTextEditorSizing: StoryFn = createStory(html`
```

Comment:
Should we also add a case that ensures the editor grows with its content? (I haven't checked if that's covered by a unit test).

Reply:
We are planning to add the test case when we support the `fit-to-content` attribute. It will be added in future PR.

However, we have the test in [this PR](https://github.com/ni/nimble/pull/1424) where we have enabled the support to set the markdown content to the editor. 
![image](https://github.com/ni/nimble/assets/123377523/05343ddf-740f-4e44-adbc-2b55fd4cdd8f)

We will cover more detailed test cases when we add the `fit-to-content` attribute.